src/lib/draw-and-pick.js

```diff
@@ -55,6 +55,57 @@ export function drawLayers({layers, pass}) {
     ${visibleCount} visible, ${layers.length} total`);
 }
 
+export function queryLayers(gl, {
```

Comment:
A short comment header would be nice. No need to list all the params but a sentence that explains what this function is supposed to do (in particular a reader would be curious to see what this does compared to all other similar functions is this file).